starboard/android/shared/media_is_key_system_supported.cc

```diff
@@ -19,23 +19,25 @@
 #include "starboard/android/shared/media_capabilities_cache.h"
 #include "starboard/android/shared/media_common.h"
 #include "starboard/media.h"
-#include "starboard/shared/starboard/media/mime_type.h"
 #include "starboard/string.h"
 
-bool SbMediaIsSupported(SbMediaVideoCodec video_codec,
-                        SbMediaAudioCodec audio_codec,
-                        const char* key_system) {
+using starboard::shared::starboard::media::MimeType;
+
+bool SbMediaIsKeySystemSupported(SbMediaVideoCodec video_codec,
+                                 SbMediaAudioCodec audio_codec,
+                                 MimeType* mime_type,
```

Comment:
It's not obvious how `mime_type` is used in this function, is it possible for you to provide some details on it.